README.md

```diff
@@ -1,9 +1,22 @@
-# JInput
-
 [![Maven Central](https://img.shields.io/maven-central/v/net.java.jinput/coreapi.svg)](https://maven-badges.herokuapp.com/maven-central/net.java.jinput/coreapi)
 [![Javadocs](http://www.javadoc.io/badge/net.java.jinput/coreapi.svg)](http://www.javadoc.io/doc/net.java.jinput/coreapi)
 
-Library for access to input devices.
+JInput is a cross-platform Java API designed for the discovery and polling of various input devices, encompassing
+keyboards, mice, joysticks, and gamepads.
+
+The supported range of devices is contingent on the underlying operating system. JInput utilizes
+[DirectInput](https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ee416842(v=vs.85)) on Windows,
+`/dev/input/event*` [device nodes](https://wiki.debian.org/device_node) on Linux, and also has support for Mac OS X.
+
+Introduced in 2003, JInput is part of a suite of open-source technologies ([JOGL](https://jogamp.org/jogl/www/),
+[JOAL](https://jogamp.org/joal/www/), JInput) initiated by the Game Technology Group at Sun Microsystems. Its
+development was led by members of the [JSR 134](https://jcp.org/en/jsr/detail?id=134) expert group, focusing on Java
+gaming.
```

Comment:
It would be interesting to hear more about its history and how you (@Endolf) came to host and maintain it here. This could be split into a `History` section as well, as not to clutter the top of the readme